Doc/library/datatypes.rst

```diff
@@ -5,13 +5,14 @@ Data Types
 **********
 
 The modules described in this chapter provide a variety of specialized data
-types such as dates and times, fixed-type arrays, heap queues, synchronized
-queues, and sets.
+types such as dates and times, fixed-type arrays, heap queues, double-ended
```

Comment:
Description of synchronized queues (the queue module) was moved to the Concurrent Execution section of the Library Reference. Mention double-ended queues instead, which are actually described in this section.